.github/CONTRIBUTING.md

```diff
@@ -77,7 +79,8 @@ When developing or maintaining community components, they typically undergo the
 
 ## Designer guide
 
-Before transforming designs into code, they must be audited.  
+Before transforming designs into code, they must be audited. Please speak to your team's DPA designer representative for more information.
```

Comment:
Replace `DPA designer representative` with `design lead` since this doc is largely for core.